modules/ding_library/ding_library.info

```diff
@@ -51,6 +51,7 @@ features[field_base][] = field_ding_library_mail
 features[field_base][] = field_ding_library_phone_number
 features[field_base][] = field_ding_library_slug
 features[field_base][] = field_ding_library_title_image
+features[field_base][] = field_hide_news_and_events
```

Comment:
It's weird that this is from culture. It was/is already in easyOPAC - but I think it was moved to separate module?

Reply:
This row _should_ exist, as the module defines this field via exports. Perhaps, it was lost during certain release.